validator/src/cli.rs

```diff
@@ -556,6 +556,12 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .hidden(hidden_unless_forced())
                 .help("Disable reporting of OS disk statistics.")
         )
+        .arg(
+            Arg::with_name("no_metrics_sanity_check")
```

Comment:
what's the usecase here? seems like the logic will already gracefully handle joining a public cluster with a private db

Reply:
Logging to a db on a private host named `tds` for mainnet? 🤡

@t-nelson rip it out?

Reply:
yeah let's start w/o it and see if there's demand.  these sanity check skipping flags have hurt us more than they've helped in the past

Reply:
🪓